subcommittees/mentoring/roles.md

```diff
@@ -0,0 +1,111 @@
+# Mentoring Subcommittee Roles Roles and Responsibilities
+
+## Roles
+
+The members of the Mentoring Subcommittee may have particular roles within the 
+functioning of the subcommittee. These roles include:
+
+* [Chair](#chair)
+* [Vice Chair](#vicechair)
+* [Secretary](#secretary)
+
+### Chair
+
+The Chair is meant to organize and facilitate the activities of the Committee.
+The Chair agrees to:
+
+1.  Organize/schedule meetings at the agreed-upon frequency.
+
+2.  Ensure that an agenda is planned and circulated in a timely
+    fashion.
+
+3.  Lead meetings fairly and transparently (or delegate meeting
+    leadership when absent).
+
+4.  Serve as the primary liaison for communication between the
+    community and the Mentoring Subcommittee members.
+
+### Vice Chair
+
+The Vice Chair is meant to support the duties of the chair. In particular, the 
+Vice Chair should adopt the responsibilities of the Chair when the Chair is 
+temporarily unable to perform their duties (e.g., absent from a meeting, 
+stepping down during a vote temporarily due to conflict of interest). The Vice
+Chair agrees to:
+
+1.  Adopt the duties and authority of the Chair at the request of the Chair or
+    by vote of the Committee.
+
+2.  Act as the primary liaison between the Steering Committee and the Subcommittee.
+
+### Secretary
+
+The Secretary is meant to maintain records of a non-financial nature. The Secretary agrees to:
+
+1. Facilitate the formation of meeting minutes.
+
+3. Archive and publish the meeting minutes [in the GitHub repository][minutes-repository].
+
+4. Manage, maintain, and facilitate other record keeping tasks where necessary.
+
+## Operations
+
+### Meeting
+
+1. Chair creates agenda for the meeting with one week in advace.
+
+   The agenda can be created on http://pad.software-carpentry.org/.
+
+2. Chair creates calendar event for the meeting with one week in advance.
+
+   The calendar event can be created on http://calendar.google.com/.
+
+3. Secretary takes notes during the meeting.
+
+4. Secretary creates issues for action itens on GitHub [repository][].
```

Comment:
“itens” → “items”.  And maybe shuffle to “Secretary creates [GitHub issues][issues] for action items.” to be more compact.

[issues]: https://github.com/swcarpentry/board/issues